pom.xml

```diff
@@ -34,10 +36,18 @@
             <id>zregvart</id>
             <name>Zoran Regvart</name>
         </developer>
+        <developer>
+            <id>bpedman</id>
+            <name>Brandon Pedersen</name>
+        </developer>
     </developers>
 
     <properties>
         <maven-release-plugin.version>2.5.1</maven-release-plugin.version>
+
+        <deps.apache-sshd.version>0.12.0</deps.apache-sshd.version>
+        <deps.jsch.version>0.1.52</deps.jsch.version>
+        <deps.ssh-credentials-plugin.version>2.0-SNAPSHOT</deps.ssh-credentials-plugin.version>
```

Comment:
Since JSch specific authenticator classes are actually in different package than it was in "ssh-credentials-plugin", this can be even lowered to current version - 1.12